reference/tools/user-guide.md

```diff
@@ -6,184 +6,271 @@ aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incre
+- Input data source:
+    - The output file of Mydumper
+    - CSV file
+- Supported TiDB versions: v2.1 or later
+- Kubernetes support: Yes. See [Quickly restore data into a TiDB cluster in Kubernetes using TiDB Lightning](/tidb-in-kubernetes/maintain/lightning.md) for details.
 
-#### Backup and restore tool BR
+#### BR (beta)
 
-[BR](/reference/tools/br/br.md) is a command-line tool used for distributed data backup and restoration for a TiDB cluster. Compared with Mydumper and Loader, BR allows you to finish backup and restore tasks with greater efficiency in scenarios of huge data volume.
+[BR](/reference/tools/br/br.md) (Backup & Restore) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with Mydumper/loader, BR is more suitable for scenarios of huge data volume.
 
 The following are the basics of BR:
 
-- [Types of backup files](/reference/tools/br/br.md#types-of-backup-files): The SST file and the `backupmeta` file.
-- Compatibility: Compatible with TiDB v3.1 and v4.0 versions.
-- Kubernetes: Supported. Relevant documents are on the way.
+- Input data source: The output file of BR
+- Supported TiDB versions: v3.1 or later
+- Kubernetes support: Yes. The document is WIP.
 
-#### Incremental and full import tool TiDB Data Migration
+#### TiDB Loader (to be deprecated)
 
-[TiDB Data Migration (DM)](/reference/tools/data-migration/overview.md) is an tool used for data migration from MySQL/MariaDB into TiDB. It supports both the full and incremental data replication.
+> **Note:**
+> 
+> TiDB Loader is to be deprecated and replaced with [Lightning](/reference/tools/tidb-lightning/tidb-backend.md#migrating-from-loader-to-tidb-lightning-tidb-back-end).
 
-The following are the basics of DM:
+[TiDB Loader](/reference/tools/loader.md) is a lightweight full-data importing tool for TiDB. It reads the output file of Mydumper and loads the data into TiDB.
 
-- Input: Full data and binlog data of MySQL/MariaDB.
-- Output: SQL statements written to TiDB.
-- Compatibility: Compatible with all TiDB versions.
-- Kubernetes: In development.
+The following are the basics of Loader:
 
-#### Full import tool Loader (Stop maintenance, not recommended)
+- Input data source: Mydumper’s output file
+- Supported TiDB versions: all versions
+- Kubernetes support: Yes. See [Backup and restore](/tidb-in-kubernetes/maintain/backup-and-restore.md) for details.
 
-[Loader](/reference/tools/loader.md) is a lightweight full data import tool. Data is imported into TiDB in the form of SQL statements. Currently, this tool is gradually replaced by [TiDB Lightning](#full-import-tool-tidb-lightning), see [TiDB Lightning TiDB-backend Document](/reference/tools/tidb-lightning/tidb-backend.md#migrating-from-loader-to-tidb-lightning-tidb-backend).
+### Incremental data import tools
 
-The following are the basics of Loader:
+#### TiDB Data Migration
```

Comment:
In fact DM can backup and restore the full data and replicate incremental data from MySQL, should we add a new category for it?

Reply:
PTAL @GregoryIan @shenli